src/encoded/static/components/genome_browser.js

```diff
@@ -359,9 +359,13 @@ class GenomeBrowser extends React.Component {
     }
 
     drawTracks(container) {
+        const highlightLocation = Math.floor((this.state.x1 - this.state.x0) / 2) + this.state.x0;
```

Comment:
Mathematically the same as avg(x1, x0) (not that JS has an average function, surprisingly). Would it be clearer that this averages if you have `Math.floor((this.state.x1 + this.state.x0) / 2)`, or does the addition of x0 outside Math.floor have an important effect?

Reply:
Oh yes, I could write it like this. I'm not entirely sure why I wrote it the other way.